R/zchunk_L113.ghg_an_R_S_T_Y.R

```diff
@@ -35,50 +38,98 @@ module_emissions_L113.ghg_an_R_S_T_Y_DISABLED <- function(command, ...) {
     GCAM_sector_tech <- get_data(all_data, "emissions/mappings/GCAM_sector_tech")
     L107.an_Prod_Mt_R_C_Sys_Fd_Y <- get_data(all_data, "L107.an_Prod_Mt_R_C_Sys_Fd_Y")
     L103.ghg_tgmt_USA_an_Sepa_F_2005 <- get_data(all_data, "L103.ghg_tgmt_USA_an_Sepa_F_2005")
-    EDGAR_CH4 <- get_data(all_data, "emissions/EDGAR_CH4")
-    EDGAR_N2O <- get_data(all_data, "emissions/EDGAR_N2O")
+    EDGAR_CH4 <- get_data(all_data, "emissions/EDGAR/EDGAR_CH4")
+    EDGAR_N2O <- get_data(all_data, "emissions/EDGAR/EDGAR_N2O")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Computing unscaled emissions by country and technology
+
+      L107.an_Prod_Mt_R_C_Sys_Fd_Y %>%
+      rename( xyear = year ) %>%
+      rename( production = value ) %>%
+      group_by(GCAM_commodity, system, feed) %>%
+      left_join(GCAM_sector_tech, by = c("GCAM_commodity" = "sector", "system" = "fuel", "feed"="technology")) %>%
+      select( GCAM_region_ID, GCAM_commodity, system, feed, xyear, production, EPA_agg_sector ) %>%
+      repeat_add_columns(tibble::tibble(Non.CO2 = c("N2O_AGR", "CH4_AGR"))) %>%  # Add Gas Name
+      group_by(EPA_agg_sector) %>%
+      left_join(L103.ghg_tgmt_USA_an_Sepa_F_2005, by = c("EPA_agg_sector" = "sector")) %>%  # match in emissions factors
+      mutate(epa_emissions = production * ch4_em_factor) %>%  # compute unscaled emissions
+      select(-fuel) %>%
+      na.omit() %>%
+      group_by( GCAM_commodity, system, feed) %>%
+      left_join(GCAM_sector_tech, by =c("GCAM_commodity" = "sector", "system" = "fuel", "feed"="technology" )) %>%
+      select( -IIASA_sector, -EPA_MACC_Sector, -BCOC_agg_fuel, -BCOC_agg_sector, -RCP_agg_sector, -EPA_agg_fuel_ghg,
+              -EPA_agg_fuel, -supplysector, -subsector, -stub.technology, -EPA_agg_sector.y) ->
+      L113.ghg_tg_R_an_C_Sys_Fd_Yh.mlt
+
+    # Aggregate by sector and region
+    L113.ghg_tg_R_an_C_Sys_Fd_Yh.mlt %>%
+      group_by(GCAM_region_ID, Non.CO2,EDGAR_agg_sector, xyear) %>%
```

Comment:
There shouldn't be an `xyear`. If there is, because e.g. this is `temp-data-inject` data, please deal with it up top when reading it.